package.json

```diff
@@ -934,7 +939,7 @@
 		"socket.io": "^1.7.3",
 		"socket.io-client": "^1.7.3",
 		"underscore": "^1.8.3",
-		"vscode-azureextensionui": "0.26.5",
```

Comment:
When you do these PRs in the future - can you add comments in the description saying what breaking changes there were and if they affect this extension - especially in cases where you're bumping past multiple breaking changes? For example:
> 0.27 didn't affect this extension and 0.28 was about the output channel

or

> 0.27 was about LocationListStep and 0.28 was about the output channel

Reply:
You got it boss.